core/domain/html_cleaner_test.py

```diff
@@ -20,16 +20,18 @@
 
 from core.domain import html_cleaner
 from core.tests import test_utils
+from typing import List, Tuple
+from typing_extensions import Final
```

Comment:
I think declaring the test data variables as Final is unnecessary and would run fine without it.

Reply:
okay, done